src/host/ut_host/TextBufferTests.cpp

```diff
@@ -1929,27 +1766,6 @@ void TextBufferTests::ResizeTraditionalHighUnicodeColumnRemoval()
     _buffer->ResizeTraditional(trimmedBufferSize);
 }
 
-void TextBufferTests::TestBurrito()
-{
-    til::size bufferSize{ 80, 9001 };
-    UINT cursorSize = 12;
-    TextAttribute attr{ 0x7f };
-    auto _buffer = std::make_unique<TextBuffer>(bufferSize, attr, cursorSize, false, _renderer);
-
-    // This is the burrito emoji: 🌯

```

Comment:
goodbye sweet burrito test

Reply:
We have more complex tests in its place now (just below this one IIRC). 😌